docs/learn/Adding Interactivity/queueing-a-series-of-state-updates.mdx

```diff
@@ -0,0 +1,195 @@
+import { Code } from '../../../src/components/Code';
+import { Card } from '../../../src/components/Card';
+
+# 일련의 상태 업데이트 저장하기
+
+state 변수를 세팅하는 것은 또 다른 렌더링을 저장합니다. 하지만 가끔 여러분은 다음 렌더링을 저장하기전에 다수의 명령을 실행하고 싶을 수 있습니다.
+이것을 하기위해서는, React가 state 업데이트를 어떻게 일괄처리하는지에 대해 이해하는 것이 도움이 됩니다.
+
+:::note You will learn
+
+- "일괄 처리(batching)"가 의미하는 바와 이것을 React가 다수의 state 업데이트를 위해 어떻게 활용하는지
+- 한 state 변수에 순차적으로 업데이트들을 적용하는 방법
+
+:::
+
+## React는 state 업데이트를 일괄처리(batch)합니다
+
```

Comment:
You might expect that clicking the “+3” button will increment the counter three times because it calls setNumber(number + 1) three times:
문구가 빠져 있습니다.